client/rest-high-level/src/test/java/org/opensearch/client/RankEvalIT.java

```diff
@@ -158,7 +158,8 @@ public void testMetrics() throws IOException {
 
             RankEvalRequest rankEvalRequest = new RankEvalRequest(spec, new String[] { "index", "index2" });
             RankEvalResponse response = execute(rankEvalRequest, highLevelClient()::rankEval, highLevelClient()::rankEvalAsync);
-            assertEquals(expectedScores[i], response.getMetricScore(), Double.MIN_VALUE);
+            double delta = 0.0001; // Set a practical
```

Comment:
Could you please introduce constant to `OpenSearchRestHighLevelClientTestCase`, fe

```
static final double DOUBLE_DELTA = 0.000001;
```

and use it here? I suspect we may run into other cases